### ISSUE TITLE
Unable to capture 3D anymore after updating Chrome to version 85

### ISSUE THREAD

**Issue by LordVulkan** [eliemichel/MapsModelsImporter#74]
Hi,

Today I updated Google Chrome to version 85, now Renderdoc captures the whole window instead of 3D buffer (or so I think, the capture shows the whole browser UI).

I can still import the capture to Blender(recently updated to 2.90), and the result is this mess:

![Mess1](https://user-images.githubusercontent.com/562386/91757646-f71ec280-ebce-11ea-8c8f-54d8949534e1.png)
![Mess2](https://user-images.githubusercontent.com/562386/91757650-f7b75900-ebce-11ea-95a8-fd2b1dd7537b.png)

The problem is that even after reinstalling an older version of Chrome, I can't make it work. I have tried with Chrome 84 and Chrome 81, and I'm pretty sure the version that worked before wasn't older than that.

Any clue of what I may be doing wrong?

Thanks.

EDIT: Attached rdc file.

https://mega.nz/file/898FELDI#VT8MRSsZc0eyxBrSiQvaAf_60upo2dqXfCb5RIaPGo4

**Comment by eliemichel** [comment]
Hi,
I am unfortunately not able to open your rdc file, so I'll need your help for troubleshooting. Could you:

 - Try a few older releases: https://github.com/eliemichel/MapsModelsImporter/releases
 - Turn on extra debug info in the add-on's preferences and copy here the **full** log (not a screenshot, text copy)

Thanks.

**Comment by LordVulkan** [comment]
I tried v0.3.0 with same results, I wiil try older versions this weekend.

Here you have the log after enabling debug info.

Thanks.

[BlenderRdcTestLog.txt](https://github.com/eliemichel/MapsModelsImporter/files/5171848/BlenderRdcTestLog.txt)

**Comment by eliemichel** [comment]
Previous versions had a bug in some cases with Google Earth captures, leading to similar looking results, maybe try the fix: https://github.com/eliemichel/MapsModelsImporter/releases/tag/v0.3.2

**Comment by e3i6** [comment]
"Bonjour,"
I'm also failing to capture more than 4 draw calls. So the plugin may work but I can't give it something to work on. I tried RenderDoc 1.10, 1.9, 1.5, on Chrome 86.0.4240.183, capture with or without delay, always moving the 3d google map view, many many times. I managed to download 80 after reading this issue but any without success.

I can capture views with RenderDoc, but they "empty" (4 draw calls) and the saved rdc doesn't open in Blender. FAQ didn't help... yet,

Any idea is welcome...
